packages/main/test/specs/Input.spec.js

```diff
@@ -671,3 +671,37 @@ describe("Input arrow navigation", () => {
 		assert.strictEqual(await valueStateHeader.getAttribute("focused"), null, "Value state header is not focused");
 	});
 });
+
+describe("XSS tests for suggestions", () => {
+	it("add suggestion item with XSS", async () => {
+		await browser.url(`http://localhost:${PORT}/test-resources/pages/Input.html`);
+
+		const btn = await $("#xss-btn");
+		const span = await $("#xss-result");
+
+		await btn.click();
+
+		assert.strictEqual(await span.getText(), "NO XSS", "No XSS issues found")
+	});
+
+	it("Tests dangerous items highlighting", async () => {
+		await browser.url(`http://localhost:${PORT}/test-resources/pages/Input.html`);
+
+		const input = await $("#xss-input");
+
+		await input.keys("[");
```

Comment:
How about you try just the simple type the letter "a" and then make sure that one of the html suggestions has a text that includes an ampersand.

You should make sure your expectations have the actual `<b>` tag wrapping the search, though. And make sure that the search of a letter "a" does not break up HTML entities that happen to contain "a"!